### ISSUE TITLE
Are you sure you need multiple watchers?

### ISSUE THREAD

**Issue by Nashet** [aekrylov/vic2_economy_analyzer#11]
Hi, I was trying to implement storing path to history (watcher) file in path.txt here -  #9 
But I failed once I realized that you have several watchers at once (And I tried to write value in table instead of file prompt dialog)

So, are you sure you need multiple watchers? It would be nice to store path to history (watcher) file in path.txt, cause it irritates to select it each time you launch tool.

**Comment by aekrylov** [comment]
You're right, multiple watchers running simultaneously are useless. I haven't really thought about that earlier :) So I think we can save pah to the last selected file
